tests/no-replaceAll.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const rule = require('../rules/no-replaceAll')
+const RuleTester = require('eslint').RuleTester
+
+const error = '$.replaceAll is potentially dangerous'
+
+const ruleTester = new RuleTester()
+ruleTester.run('no-replaceAll', rule, {
+  valid: [
+    'replaceAll()',
+    '[].replaceAll()',
+    'div.replaceAll()',
+    'div.replaceAll',
+    '$div.replaceAll()',
+    '$div.first.replaceAll()'
+  ],
+  invalid: [
+    {
+      code: '$("div").replaceAll("<script>")',
+      errors: [{message: error, type: 'CallExpression'}]
+    },
+    {
+      code: '$div.replaceAll("<script>")',
+      errors: [{message: error, type: 'CallExpression'}]
+    },
+    {
+      code: '$("div").first().replaceAll("<script>")',
+      errors: [{message: error, type: 'CallExpression'}]
+    },
+    {
+      code: '$("div").append($("input").replaceAll("<script>"))',
+      errors: [{message: error, type: 'CallExpression'}]
+    }
+  ]
```

Comment:
`replaceAll` method is different than than the rest of unsafe methods. Calee and parameter are reversed, see https://api.jquery.com/replaceAll/. 

```js
$( "<script>" ).replaceAll( "div" );
```

In order to better understand this, try to create XSS-vulnerable code with `replaceAll`.

- Please fix these tests based on https://api.jquery.com/replaceAll

Reply:
Ohh alright, will check more on this and update accordingly.